docs-kits/kits/Behaviour Twin MDP Kit/page_development-view.md

```diff
@@ -0,0 +1,324 @@
+---
+id: page_development-view
+title: Development View
+description: Model Based Development and Data Processing Kit
+---
+
+![Model Based Development and Data Processing Kit banner](/img/kit-icons/behaviour-mdp-kit-icon.svg)
+
+### Model Based Development and Data Processing Kit
+
+## MDP Development Video
+
+This video demonstrates the necessary software application functionality for implementing the MDP Use Cases:
```

Comment:
The development view should remain vendor-neutral. Therefore, please (1) move the video and (2) all Siemens related description to the success story or re-write the section in a vendor-neutral way. Thanks!

Reply:
OK, we removed the video and other Siemens related content from the Development View and moved it to the Success Stories section.